packages/regenerator-runtime/runtime.js

```diff
@@ -102,8 +102,9 @@ var runtime = (function (exports) {
 
   var Gp = GeneratorFunctionPrototype.prototype =
     Generator.prototype = Object.create(IteratorPrototype);
```

Comment:
wouldn't this line also perhaps trigger the same issue, since `.prototype` is also locked down?

Reply:
I believe this is covered by the added tests. function-syntax classes have their own writable `prototype` property, so it is not affected by the override mistake
```js
function Animal () {}
Reflect.getOwnPropertyDescriptor(Animal, 'prototype')
//=> Object { value: {…}, writable: true, enumerable: false, configurable: false }
```